src/stability.md

```diff
@@ -85,7 +85,7 @@ To stabilize a feature, follow these steps:
    - Add the appropriate labels: `@rustbot modify labels: +T-libs`.
    - Link to the tracking issue and say "Closes #XXXXX".
 
-You can see an example of stabilizing a feature at [#75132](https://github.com/rust-lang/rust/pull/75132).
+You can see an example of stabilizing a feature at [#81656](https://github.com/rust-lang/rust/issues/81656).
```

Comment:
The section talks about how we create a PR so we should link to the PR, not the issue.

Reply:
I've added a link to both and clarified the wording so it points to all steps of the process.